webui/tests/specs/02-experiments.spec

```diff
@@ -24,13 +24,13 @@ Specification to test the experiment list page.
   |Kill               |false   |
 * Toggle all table row selection
 
-## Filter experiments by archived
+// ## Filter experiments by archived
```

Comment:
Is there a reason we are using `//` instead of `#`?

Reply:
for gauge tests, it uses a special markdown and `#` is treated as a title/section

Reply:
Thank you!